### PR TITLE
Prepare for C++ attestation release

### PR DESCRIPTION
**Pull request by LarryOsterman**
# Pull Request Checklist

Please leverage this checklist as a reminder to address commonly occurring feedback when submitting a pull request to make sure your PR can be reviewed quickly:

See the detailed list in the [contributing guide](https://github.com/Azure/azure-sdk-for-cpp/blob/main/CONTRIBUTING.md#pull-requests).

- [X] [C++ Guidelines](https://azure.github.io/azure-sdk/cpp_introduction.html)
- [X] Doxygen docs
- [X] Unit tests
- [X] No unwanted commits/changes
- [X] Descriptive title/description
  - [X] PR is single purpose
  - [ ] Related issue listed
- [X] Comments in source
- [X] No typos
- [X] Update changelog
- [X] Not work-in-progress
- [X] External references or docs updated
- [X] Self review of PR done
- [X] Any breaking changes?

### CODE REVIEW

File: sdk/attestation/azure-security-attestation/CHANGELOG.md [hunk 1]
```diff
@@ -1,12 +1,6 @@
 # Release History
 
-## 1.0.0.beta.1 (Unreleased)
+## 1.0.0-beta.1 (2022-04-05)
 
 ### Features Added
 * Attestation Package creation
```
Comment (vhvb1989) [inline sdk/attestation/azure-security-attestation/CHANGELOG.md:6]: We use `-` instead of `*`
And there should be one empty line between a tittle and the bullets
```suggestion

- Attestation Package creation
```

Comment (LarryOsterman) [reply sdk/attestation/azure-security-attestation/CHANGELOG.md:6]: Should I also make this change in template, since that's where I copied this from?

File: sdk/template/azure-template/CHANGELOG.md [hunk 2]
```diff
@@ -2,103 +2,103 @@
 
 ## 1.0.0 (2021-06-03)
 
-* Validate template project ahead of release
+- Validate template project ahead of release
```
Comment (vhvb1989) [inline sdk/template/azure-template/CHANGELOG.md:5]: I would move this to another PR, so it's not smuggled within attestation release.
Thank you for changing it for template.

**Comment by LarryOsterman** [comment]
Ping. This should be pretty uncontroversial :).

**Review by vhvb1989 — APPROVED** [review]
Didn't approve before because these sort of changelog PRs for release are not merged until the release day, as we want the commit to be the one for the release tag.
